examples/docker-example/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM thinktopic/cortex-base
+MAINTAINER ThinkTopic
+
+ENV service docker-example
+ADD target/uberjar/${service}.jar /srv/${service}.jar
```

Comment:
small thing - but when I run `lein uberjar` it creates the docker-example.jar in `target` not `target/uberjar`